lib/msal-core/src/telemetry/CacheEvent.ts

```diff
@@ -0,0 +1,29 @@
+import TelemetryEvent from "./TelemetryEvent";
+import { EVENT_NAME_PREFIX } from "./TelemetryConstants";
+
+export const CACHE_EVENT_TYPES = {
+    TokenCacheLookup: `${EVENT_NAME_PREFIX}token_cache_lookup`,
+    TokenCacheWrite: `${EVENT_NAME_PREFIX}token_cache_write`,
+    TokenCacheBeforeAccess: `${EVENT_NAME_PREFIX}token_cache_before_access`,
+    TokenCacheAfterAccess: `${EVENT_NAME_PREFIX}token_cache_after_access`,
+    TokenCacheBeforeWrite: `${EVENT_NAME_PREFIX}token_cache_before_write`,
+    TokenCacheDelete: `${EVENT_NAME_PREFIX}token_cache_delete`,
+    TokenType: `${EVENT_NAME_PREFIX}token_type`
+};
+
+export const TOKEN_TYPES = {
+    AT: "at",
```

Comment:
This is being really nitpicky, but I'd prefer "access" instead of "at", just to be clear.

Reply:
I think this is what dot net is setting in their telemetry, keeping for consistency. I appreciate nit picks anyway :p